test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -1,26 +1,25 @@
 'use strict';
 /*
  * This test is a regression test for joyent/node#8900.
+ *
+ * Node.js 0.10.34 contains the bug that this test detects.
+ * Node.js 0.10.35 contains the fix.
+ *
+ * Due to the introduction of ES6-isms, comment out `'use strict'` and
+ * `require('../common');` to run the test on Node.js 0.10.x.
```

Comment:
Shouldn't be necessary. If testing is needed, you could probably just undo the timers fix: https://github.com/nodejs/node-v0.x-archive/pull/8906/files#diff-0a5d4868b2b9b17cf9e2c11f1bd1311eL296